partcad-ide-vscode/bundled/tool/lsp_jsonrpc.py

```
@@ -244,7 +248,9 @@
     data = rpc.receive_data()
 
     if data["id"] != msg_id:
-        return RpcRunResult("", f"Invalid result for request: {json.dumps(msg, indent=4)}")
+        return RpcRunResult(
+            "", f"Invalid result for request: {json.dumps(msg, indent=4)}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)